namespaces/cloud-platform-test-1.k8s.integration.dsd.io/traefik/traefik-ui.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
The traefik web UI shouldn't be exposed for various reasons as discussed, we can always reach it with `kubectl port-forward`.